src/WaitForReact.js

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { throttle } from 'lodash';
+import changeProps from 'react-change-props';
+import script from './inline-script.raw.js';
+
+const pickScriptState = () =>
+    typeof window !== 'undefined' && Array.isArray(window.__WAIT_FOR_IT__) ? window.__WAIT_FOR_IT__.shift() : undefined;
```

Comment:
Maybe use `__REACT_WAIT_FOR_IT__`? Just an opinion. Either is fine.
But having the `REACT_*` prefix makes it more obvious it's for a react package.

```suggestion
    typeof window !== 'undefined' && Array.isArray(window.__REACT_WAIT_FOR_IT__) ? window.__REACT_WAIT_FOR_IT__.shift() : undefined;
```

Reply:
Fixed!